File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int l=0;
        int r=numbers.length-1;

        while(r>l)
        {
            if(numbers[r]+ numbers[l] == target)
            return new int[]{l+1,r+1};
            else if(numbers[r]+ numbers[l] > target)
            r--;
            else
            l++;
        }

        return new int[2];
    }
}

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        l,r= 0,len(numbers)-1

        while(l<r):
            sum = numbers[l]+numbers[r]
            if sum==target:
                return [l+1,r+1]
            elif sum > target:
                r -=1
            elif sum < target:
                l +=1
        